sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/api/CaseSteps.java

```diff
@@ -17,17 +17,116 @@
  */
 package org.sormas.e2etests.steps.api;
 
+import static com.google.common.truth.Truth.assertThat;
+
 import cucumber.api.java8.En;
 import javax.inject.Inject;
+import org.sormas.e2etests.enums.APITestData.ErrorMessages;
 import org.sormas.e2etests.helpers.api.CaseHelper;
+import org.sormas.e2etests.helpers.api.CommunityHelper;
+import org.sormas.e2etests.helpers.api.CountryHelper;
+import org.sormas.e2etests.helpers.api.PersonsHelper;
 import org.sormas.e2etests.pojo.api.Case;
 import org.sormas.e2etests.services.api.CaseApiService;
+import org.sormas.e2etests.services.api.PostCaseBodyService;
+import org.sormas.e2etests.services.api.PostPersonBodyService;
 import org.sormas.e2etests.state.ApiState;
+import org.sormas.e2etests.state.BodyResources;
 
 public class CaseSteps implements En {
 
   @Inject
-  public CaseSteps(CaseHelper caseHelper, CaseApiService caseApiService, ApiState apiState) {
+  public CaseSteps(
+      CaseHelper caseHelper,
+      PersonsHelper personHelper,
+      ApiState apiState,
+      CaseApiService caseApiService,
+      CommunityHelper communityHelper,
+      CountryHelper countryHelper,
+      BodyResources bodyResources,
+      PostCaseBodyService bodyService,
+      PostPersonBodyService personBodyService) {
+
+    Given(
+        "I create a person",
```

Comment:
please rename this to make it more clear that we're using API calls   (also, we already have this in place)